modules/shared.py

```diff
@@ -158,3 +162,10 @@ def str2bool(v):
 
 def is_chat():
     return args.chat
+
+
+def clone_and_import(repo_url, local_path, module_name):
+    subprocess.run(["git", "clone", repo_url, local_path])
+    sys.path.insert(0, local_path)
+    module = importlib.import_module(module_name)
+    return module
```

Comment:
(Minor) make sure to leave a newline @ end of file (you can see the lil red warning sign github puts here about it)